src/libs/actions/Report.js

```diff
@@ -398,26 +398,6 @@ function setLocalIOUReportData(iouReportObject) {
     Onyx.merge(iouReportKey, iouReportObject);
 }
 
-/**
- * Remove all optimistic actions from report actions and reset the optimisticReportActionsIDs array. We do this
- * to clear any stuck optimistic actions that have not be updated for whatever reason.
- *
- * @param {Number} reportID
- */
-function removeOptimisticActions(reportID) {
-    const actionIDs = optimisticReportActionIDs[reportID] || [];
-    const actionsToRemove = _.reduce(actionIDs, (actions, actionID) => ({
-        ...actions,
-        [actionID]: null,
-    }), {});
-    Onyx.merge(`${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${reportID}`, actionsToRemove);
-
-    // Reset the optimistic report action IDs to an empty array
-    Onyx.merge(`${ONYXKEYS.COLLECTION.REPORT}${reportID}`, {
-        optimisticReportActionIDs: [],
-    });
-}
```

Comment:
This is kind of an MVP solution that I'm removing now. If we leave it in each time the report actions are fetched we will remove any "optimistic actions". That includes any pending actions with errors which breaks the new design. It was always the plan to remove this code so if we end up with a problem related to "stuck actions" then we'll just need to approach it another way. Though, I'm not sure if "stuck comments" are really a thing anymore.